src/pds/ingress/client/pds_ingress_client.py

```diff
@@ -103,8 +107,12 @@ def _perform_ingress(ingress_path, node_id, prefix, api_gateway_config):
     except Exception as err:
         # Only log the error as a warning, so we don't bring down the entire
         # transfer process
-        logger.warning(f"{trimmed_path} : Ingress failed, reason: {str(err)}")
+        reason = err.response.json() if isinstance(err, requests.exceptions.HTTPError) else str(err)
+        logger.warning(f"{trimmed_path} : Ingress failed, reason: {reason}")
```

Comment:
As a reminder, f-strings are fantastic (and that might be what the "f" stands for) but when logging, you should use
```python
logger.warning("%s : Ingress failed, reason: %s", trimmed_path, reason)
```
The reason: if the category of logging is turned off, then the logger won't even bother interpolating the string—it'll just throw it out. Using an f-string makes the interpolation happen regardless—even if the logger ends up throwing it out.

Reply:
@collinss-jpl FYI reminder for the future ☝️